addons/product/models/product_template.py

```diff
@@ -381,9 +381,23 @@ def _name_search(self, name, args=None, operator='ilike', limit=100, name_get_ui
             if (not products) or (limit and (len(templates) > limit)):
                 break
 
+        searched_ids = set(templates.ids)
+        # some product.templates do not have product.products yet (dynamic variants configuration),
+        # we need to add the base _name_search to the results
+        # FIXME awa: this is really not performant at all but after discussing with the team
+        # we don't see another way to do it
+        if len(searched_ids) < limit:
+            searched_ids |= set([template_id[0] for template_id in
+                super(ProductTemplate, self)._name_search(
+                    name=name,
```

Comment:
Name seems to be a positional parameter, does that work in python ? Anyway it should be _name_search(name, args=args, ...)

Reply:
I copy/pasted that from somewhere else, should have been more careful, good catch!